config/workers.coffee

```diff
@@ -116,13 +174,34 @@ module.exports = (KONFIG, options, credentials) ->
         ]
       healthCheckURLs   : [ "http://localhost:#{KONFIG.kloud.port}/healthCheck" ]
       versionURL        : "http://localhost:#{KONFIG.kloud.port}/version"
+      kubernetes        :
+        image           : 'koding/base'
```

Comment:
We should not need the command section here. This worker definition already has it.

Reply:
Same applies for ports section too.